diff --git a/src/SendinBlueTransport.php b/src/SendinBlueTransport.php
index 11e0bd7..07ec06c 100755
--- a/src/SendinBlueTransport.php
+++ b/src/SendinBlueTransport.php
@@ -9,6 +9,8 @@ use Swift_Mime_SimpleMessage;
 use Swift_MimePart;
 use Swift_Mime_Headers_UnstructuredHeader;

+use App\Events\providerSentEmail;
+
 class SendinBlueTransport extends Transport
 {
     /**
@@ -38,6 +40,8 @@ class SendinBlueTransport extends Transport

         $res = $this->mailin->send_email($this->buildData($message));

+        event(new providerSentEmail($res, $message));
+
         if (!$res) {
             throw new SendinBlueException("Unknown error");
         }